### ISSUE TITLE
T-ORPF extension - plans ?

### ISSUE THREAD

**Issue by udf2457** [stef/pwdsphinx#19]
Thank you for your work on this.  I wanted to find out if you had any plans to expand this to cover the T-ORPF/PPSS concept discussed in your presentation ? So that K<sub>d</sub> could be stored in a redundant manner.

**Comment by stef** [comment]
yes, definitely, but it will be a backwards incompatible change. i expect a beta out in a few weeks, work on this will start in september.

**Comment by udf2457** [comment]
Fantastic news.  Thanks !

**Comment by stef** [comment]
update: all unittests succeed with the new threshold liboprf as a backend. there is no release yet, but if you wanna try it, get liboprf and the https://github.com/stef/pwdsphinx/tree/v2-nextgen branch

**Comment by stef** [comment]
the config files have changed a bit, documentation on that is still lacking though... it's all WIP but the code already works!

**Comment by joostvb-gh** [comment]
Cool, I'll try to get a v2-nextgen pwdsphinx prerelease in Debian / experimental.  stef: will get back to you on this.
